CHANGELOG.unreleased.md

```diff
@@ -35,6 +36,8 @@
 >
 > Rule of thumb: add packages on top.
 
+- @xen-orchestra/xo-upload-ova major
+- xo-vmdk-to-vhd major
```

Comment:
Is the dependency order rule respected?

Reply:
not sure. `@xen-orchestra/xo-upload-ova` depends on `xo-vmdk-to-vhd`

I never know what the right topological order is.

Reply:
Packages are released in order, therefore dependencies should go above.